.circleci/config.yml

```diff
@@ -0,0 +1,110 @@
+version: 2
+
+aliases:
+
+  - defaults: &defaults
+      docker:
+        - image: circleci/node:carbon
+      working_directory: ~/code
+  - &restore-repo-cache
+      key: shared-goodcity-repo-{{ .Environment.CIRCLE_SHA1 }}
+  - &restore-yarn-cache
+      keys:
+        - shared-goodcity-yarn-packages-{{ checksum "yarn.lock" }}
+  - &save-yarn-cache
+      key: shared-goodcity-yarn-packages-{{ checksum "yarn.lock" }}
+      paths:
+        - node_modules
+        - ~/.cache/yarn
+  - &yarn
+    |
+      yarn install --non-interactive --cache-folder ~/.cache/yarn --frozen-lockfile
+  - &restore-bower-cache
+      key: shared-goodcity-bower-components-{{ checksum "bower.json" }}
+  - &save-bower-cache
+      key: shared-goodcity-bower-components-{{ checksum "bower.json" }}
+      paths:
+        - bower_components
+
+jobs:
+  checkout_code:
+    <<: *defaults
+    steps:
+      - checkout
+      - save_cache:
+          key: shared-goodcity-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/code
+
+  package_dependencies:
+    <<: *defaults
+    steps:
+      - restore-cache: *restore-repo-cache
+      - restore-cache: *restore-yarn-cache
+      - run: yarn link
+      - run: yarn add codeclimate-test-reporter
+      - run: yarn add bower
```

Comment:
Would it work to add codeclimate-test-report and bower to the package.json/yarn lockfile so they are automatically included in the build? This should save us some precious build time due to bundle caching.